R/normalize_SE.R

```diff
@@ -164,10 +164,10 @@ normalize_SE <- function(se,
     trt_df <- data.table::as.data.table(trt_df)
     
     # Merge to ensure that the proper discard_key values are mapped.
-    all_readouts_df <- if (is.null(nested_confounders)) {
-      ref_df[trt_df, ]
-    } else {
+    all_readouts_df <- if (length(nested_confounders)) {
       ref_df[trt_df, on = nested_confounders]
+    } else {
+      ref_df[trt_df, ]
```

Comment:
Do we have any tests that check such detials?

Reply:
Added in [a5eb77a](https://github.com/gdrplatform/gDRcore/pull/98/commits/a5eb77a3ddd345d4bd2589ec1591db80c69d359e)